noether_tpp/include/noether_tpp/tool_path_planners/raster/raster_planner.h

```diff
@@ -78,7 +80,6 @@ class RasterPlanner : public ToolPathPlanner
   DirectionGenerator::ConstPtr dir_gen_;
   OriginGenerator::ConstPtr origin_gen_;
 
-private:
   /** @brief Distance between waypoints on the same raster line (m) */
   double point_spacing_;
```

Comment:
I assume making these attributes public was intentional, probably so they can be accessed by derived classes.  If you had something else in mind, let me know.